Modules/HypnokinkModule/Perks/HK_D1_Keywords.gd

```diff
@@ -0,0 +1,34 @@
+extends PerkBase
+
+func _init():
+	id = Perk.KeywordsDrawback
+	skillGroup = Skill.Hypnosis
+
+func getVisibleName():
+	return "Keyword Fixation"
+
+func getVisibleDescription():
+	return "You find your thoughts lingering on certain words as they are spoken."
+	
+func getMoreDescription():
+	return "While in "+HK_Utils.tranceIcon()+"Trance, hearing one of the keywords slightly deepens your hypnosis and increases your [color="+DamageType.getColorString(DamageType.Lust)+"]lust[/color] and arousal by a small amount."
```

Comment:
Having this whole skill tree have these fancy icons makes it inconsistent with the others x3

Reply:
I just think they're neat :] (and technically, other skill trees get to color their text instead, since they don't rely on a specific status effect)